flang-rt/include/flang-rt/CufRuntime/allocator.h

```diff
@@ -1,13 +1,13 @@
-//===-- include/flang/Runtime/CUDA/allocator.h ------------------*- C++ -*-===//
+//===-- include/flang-rt/CufRuntime/allocator.h -----------------*- C++ -*-===//
```

Comment:
I would prefer to keep CUDA here rather than CufRuntime.

Reply:
These are the files linked into a library called `CufRuntime_cuda_${CUDAToolkit_VERSION_MAJOR}`. Using `CufRuntime` seems obvious to me. Why do you prefer CUDA? Would you call the library `libCUDA.a`?

Reply:
CUDA Fortran is the name of the language.  "cuf" is a filename extension.

Reply:
As @klausler mentioned, this is the "CUDA" part of the Fortran runtime so the initial name seems fine. I don't see a place that mentions the lib name should be salted to the directory name. If that's the case then I would rather change the lib name.